taxi/forms.py

```diff
@@ -0,0 +1,54 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django import forms
+from taxi.models import Driver, Car
+
+
+class LicenseNumberValidatorMixin:
+    @staticmethod
+    def validate(license_number) -> str | ValidationError:
+        if (
+            (len(license_number) != 8)
+            or ((license_number[:3] != license_number[:3].upper())
+                or (not (license_number[:3]).isalpha()))
+            or (not license_number[3:].isnumeric())
+        ):
+            raise ValidationError("Some errors!!")
```

Comment:
Code Quality Issue: The error message in the `ValidationError` is not descriptive enough. It should explain what the error is and how to fix it. For example, you can say 'License number must be 8 characters long, start with 3 uppercase letters, and end with 5 numbers'.